src/cart/SConscript

```diff
@@ -22,6 +22,19 @@ SRC = ['crt_bulk.c', 'crt_context.c', 'crt_corpc.c',
        'crt_swim.c', 'crt_tree.c', 'crt_tree_flat.c', 'crt_tree_kary.c',
        'crt_tree_knomial.c', 'crt_hlc.c', 'crt_hlct.c']
 
+LIB_SRC = ['crt_utils.c']
+
+####################################
+# Create cart utility share objects
+####################################
+def build_utility_shared_obj():
+    """Build cart utility library"""
+
+    denv = env.Clone()
```

Comment:
(pylint-undefined-variable) Undefined variable 'env'

Reply:
pass env in